### PR TITLE
feat: add HTML standard format for ListingFormat

### PR DESCRIPTION
**Pull request by Swizbiz**
implement HTML format for slice listing and tests for this

close: #430

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, take a look

File: src/main/java/com/artipie/http/slice/ListingFormat.java [hunk 1]
```diff
@@ -48,6 +48,34 @@ enum Standard implements ListingFormat {
...
+                    "    <ul>",
+                    "%s",
+                    "    </ul>",
+                    "  </body>",
+                    "</html>"
+                ),
+                keys.stream().map(
+                    key -> String.format(
+                        "      <li><a href=\"%s\">%s</a></li>",
+                        key.string(),
```
Comment (dgarus) [inline src/main/java/com/artipie/http/slice/ListingFormat.java:74]: Method `string()` can take some resources.
Could we create a variable here and use it twice?

File: src/test/java/com/artipie/http/slice/ListingFormatTest.java [hunk 2]
```diff
@@ -68,4 +70,48 @@ void formatJsonEmptyKeys() {
             new StringIsJson.Array(new JsonContains())
         );
     }
+
+    @Test
+    void formatHtmlKeys() throws IOException {
```
Comment (dgarus) [inline src/test/java/com/artipie/http/slice/ListingFormatTest.java:75]: Unused `IOException`

**Review by dgarus — CHANGES_REQUESTED** [review]
@Swizbiz thanks!
Pay attention to my comments, please.

File: src/main/java/com/artipie/http/slice/ListingFormat.java [hunk 3]
```diff
@@ -48,6 +48,34 @@ enum Standard implements ListingFormat {
             keys -> Json.createArrayBuilder(
                 keys.stream().map(Key::string).collect(Collectors.toList())
             ).build().toString()
+        ),
+
+        /**
+         * Json format renders keys as JSON array with
```
[the remark this answers is not in the document]
Comment (Swizbiz) [reply src/main/java/com/artipie/http/slice/ListingFormat.java]: fixed

File: src/main/java/com/artipie/http/slice/ListingFormat.java (hunk 1 above)
Comment (Swizbiz) [reply src/main/java/com/artipie/http/slice/ListingFormat.java]: done

File: src/test/java/com/artipie/http/slice/ListingFormatTest.java (hunk 2 above)
Comment (Swizbiz) [reply src/test/java/com/artipie/http/slice/ListingFormatTest.java]: removed

**Comment by Swizbiz** [comment]
@dgarus please check again

**Comment by Swizbiz** [comment]
@olenagerasimova ready to merge, please check

**Review by olenagerasimova — APPROVED** [review]
@Swizbiz thanks

**Pull request merged**